dask/dataframe/accessor.py

```diff
@@ -325,6 +325,18 @@ def extractall(self, pat, flags=0):
     def __getitem__(self, index):
         return self._series.map_partitions(str_get, index, meta=self._series._meta)
 
+    @derived_from(pd.Series.str)
+    def extract(self, *args, **kwargs):
+        return self._function_map("extract", *args, **kwargs)
+
+    @derived_from(pd.Series.str)
+    def startswith(self, *args, **kwargs):
+        return self._function_map("startswith", *args, **kwargs)
+
+    @derived_from(pd.Series.str)
+    def endswith(self, *args, **kwargs):
+        return self._function_map("endswith", *args, **kwargs)
```

Comment:
[Nit] It'd be nice to add a brief comment about why these ones are explicitly defined -- even just pointing to the sphinx issue

Reply:
yep sure